core/builder/src/main/java/io/quarkus/builder/Execution.java

```diff
@@ -56,12 +68,36 @@
             done = true;
     }
 
+    Map<StepInfo, Integer> orderTopologically(Set<StepInfo> startSteps) {
+        if (!(startSteps instanceof SortedSet)) {
```

Comment:
If this value must be a `SortedSet`, then its type should be `SortedSet` rather than having all these dynamic checks around.  All this stuff is package-private I believe.

Reply:
The whole method is gone in aacdcf7